test/tests/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+yum install nodejs npm
+npm -g install phantomjs
+
+yum install yum-plugin-copr
+yum copr enable lmr/Autotest
+yum install avocado
+yum install avocado-virt
+
+# because of troubles with cockpit recompiled to /opt/ instead of /usr
+setenforce 0
+
+mkdir -p /usr/share/avocado/tests/
+
+/bin/cp -rf  * /usr/share/avocado/tests/; python checklogin.py ; cat /root/avocado/job-results/latest/job.log 
```

Comment:
So will running the test be separate from the setup code? Have you thought about how can we speed up the above setup tasks so that they do not need to run before each test?